keyboards/finindustries/tunnel75/rules.mk

```diff
@@ -0,0 +1,3 @@
+LTO_ENABLE = yes
+
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

Comment:
```suggestion
# This file intentionally left blank
```

Reply:
This change has not been made correctly.

Reply:
How should I modify it?

Reply:
Like the suggestion.

Reply:
add this?    # This file intentionally left blank